### DISCUSSION TITLE
Printers

### DISCUSSION THREAD

**Discussion by arky217** [SpiralLinux/SpiralLinux-project#386] [General]
I have several Brother usb printers, model HL-L2320D.
I have used a dozen or more of the more popular linux distros over the past few years,
including MX, Manjaro, Sparky, Peppermint, Mint, Endeavour, Ubuntu, Debian, Pop_os, Zorin, Lite, PClinuxOS, 
Q4OS, Lubuntu, and several others.

Spiral linux is the only one that automatically installs this Brother printer.
I don't have to download and install Brother printer drivers or do anything.
As soon as I turn the printer on, Spiral installs it and it is ready for printing is seconds.

On ALL the other distros, I have to install the Brother printer drivers.

This feature alone has endeared me to make Spiral my daily user.

So, how does Spiral do this ?  Why can't the other distros ?

**Comment by geckolinux** [comment]
Hi there, basically the automatic discovery on the network is thanks to the `cups-browsed` package. And to make the printer actually be able to print SpiralLinux pre-installs all the CUPS driver packages that Debian offers, plus there is a new "driverless" standard that newer printers use that is also supported.
